rtl/src/main/scala/tensil/tcu/DataFlowControl.scala

```diff
@@ -31,27 +31,12 @@ object DataFlowControlWithSize {
 }
 
 object DataFlowControl {
```

Comment:
Maybe `LocalDataFlowControl` and `LocalRouter` to clarify the difference with `HostDataFlowControl` and `HostRouter`?

Reply:
Done.